crates/biome_json_analyze/tests/specs/nursery/noDuplicateDependencies/invalid/dependencies/package.json.snap

```diff
@@ -0,0 +1,41 @@
+---
+source: crates/biome_json_analyze/tests/spec_tests.rs
+expression: package.json
+---
+# Input
+```json
+{
+  "name": "invalid-dependencies",
+  "dependencies": {
+    "foo": "",
+    "bar": "",
+    "foo": ""
+  }
+}
+
+```
+
+# Diagnostics
+```
+package.json:4:5 lint/nursery/noDuplicateDependencies ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  i The dependency "foo" is also listed under dependencies.
```

Comment:
Sorry, I'm goldplating at this point. It would be great if the message here said "The dependency "foo" is listed twice under dependencies."

Reply:
Oh, but @ematipico mentioned I should keep it simple with returning just a single couple; https://github.com/biomejs/biome/pull/7142#discussion_r2278796766